config/component.config.js

```diff
@@ -62,29 +62,41 @@ export const componentConfig = {
     youtube: {
       color: '#008000',
       model: {
-        url: ''
+        videoId: '',
+        fullscreen: true
       },
       schema: {
         fields: [
           {
             type: 'text',
-            title: 'Url',
-            model: 'url'
+            title: 'Video id',
+            model: 'videoId'
+          },
+          {
+            type: 'toggle',
+            title: 'Allow Fullscreen',
+            model: 'fullscreen'
           }
         ]
       }
     },
     embedded: {
       color: '#0000ff',
       model: {
-        url: ''
+        url: '',
+        fullscreen: true
       },
       schema: {
         fields: [
           {
             type: 'text',
             title: 'Url',
```

Comment:
Here we could also add some example video URL as a placeholder or under the input, if it's possible